frontend/src/components/terms/terms.js

```diff
@@ -13,38 +13,36 @@ const Terms = () => {
         </div>
       </div>
       <div className="agreement">
-          <div class="row">
-              <div class="col-lg-2">
+          <div >
+              <div>
               </div>
-              <div class="col-lg-8">
-                <h3>Your Agreement</h3>
-                <p>By using this Site, you agree to be bound by, and to comply with, these Terms and Conditions. If you do not agree to these Terms and Conditions, please do not use this site.</p>
-                <p>PLEASE NOTE: We reserve the right, at our sole discretion, to change, modify or otherwise alter these Terms and Conditions at any time. Unless otherwise indicated, amendments will become effective immediately. Please review these Terms and Conditions periodically. Your continued use of the Site following the posting of changes and/or modifications will constitute your acceptance of the revised Terms and Conditions and the resonableness of these standards for notice of changes. For your information, this page was last updated as of the date at the top of these terms and conditions.</p>
-                <p>
+              <div >
+                <h3 className="terms_space">Your Agreement</h3>
```

Comment:
no need for new classes just repace the class with the className keyword according to react syntax of writing class

Reply:
ok